snippets/serverless-domain-mapping.adoc

```diff
@@ -0,0 +1,8 @@
+// Text snippet included in the following files
+//
+// * serverless/security/serverless-custom-domains.adoc
+// * modules/serverless-domain-mapping-odc-admin.adoc
+
+Knative services are automatically assigned a default domain name based on your cluster configuration. For example, `<service_name>.<namespace>.example.com`. You can customize the domain for your Knative service by mapping a custom domain name that you own to a Knative service.
```

Comment:
```suggestion
Knative services are automatically assigned a default domain name based on your cluster configuration. For example, `<service_name>-<namespace>.example.com`. You can customize the domain for your Knative service by mapping a custom domain name that you own to a Knative service.
```

OpenShift Serverless sets `<service_name>-<namespace>`, which is different from upstream.